dependencies.yaml

```diff
@@ -618,8 +618,10 @@ dependencies:
           - nbsphinx
           - numpydoc
           - pandoc
-          # https://github.com/pydata/pydata-sphinx-theme/issues/1539
-          - pydata-sphinx-theme!=0.14.2
+          # Minimum version for safe parallel writing again.
```

Comment:
It seems like we may also need `sphinx>=8.1`? That is the minimum Sphinx version to get https://github.com/sphinx-doc/sphinx/pull/12888, which is referenced in https://github.com/pydata/pydata-sphinx-theme/pull/1859 (as "the bug is really in Sphinx").

Reply:
Done.